sdk/program/src/stake/instruction.rs

```diff
@@ -695,6 +695,21 @@ pub fn get_minimum_delegation() -> Instruction {
     )
 }
 
+pub mod utils {
+    /// Helper function for programs to get the actual data after calling GetMinimumDelegation
+    ///
+    /// This fn handles calling `get_return_data()`, ensures the result is from the correct
+    /// program, and returns the correct type.  Returns `None` otherwise.
+    pub fn get_minimum_delegation_data() -> Option<u64> {
```

Comment:
What would be the right/best location and/or name for this helper function?

Reply:
Good question, it needs to be somewhere where it can be pulled into a BPF program without compatibility issues. Right now I think the best place is the `stake` module in the `solana-program` crate

Reply:
Also I was hoping that this utility function would include the CPI as well

Reply:
Sorry, I didn't really acknowledge that this utility method is already where I suggested it should be haha. How about using a `tools` module to mirror how ATA provides a CPI interface? https://github.com/solana-labs/solana-program-library/tree/master/associated-token-account/program/src/tools

Reply:
Done!